docs/content/latest/manage/data-migration/ycql/bulk-import.md

```diff
@@ -100,7 +100,8 @@ INTO OUTFILE '/path/to/sample.csv' FIELDS TERMINATED BY ',';
 
 ## Import data
 
-These instructions are organized by the size of the input datasets, ranging from small (few MB of data), to medium (GB) to large (TB or larger) datasets. 
+These instructions are organized by the size of the input datasets, ranging from small (few MB of data) to large (GB
+ or larger) datasets. 
```

Comment:
Lets change this to:

...ranging from small (MBs of data) to larger datasets (GBs of data).